### PR TITLE
Export Course Grades as CSV

### PR DESCRIPTION
**Pull request by epai**
Allows student grades to be exported as a CSV file.  

Currently splits scores by assignment and "score type", where score type is either "Total", "Composition", "Checkpoint 1", or "Checkpoint 2".  

Only assignments that are visible to students and have published scores will be "checked" by default.

Note:  Exporting may take a while (1-2 hours)

### CODE REVIEW

File: server/jobs/export_grades.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+        scores['total'] = total_score
+    if scores_checker(scores, COMP_KINDS):
+        composition_score = max(score_grabber(scores, COMP_KINDS))
+        scores['composition'] = composition_score
+    return scores
+
+
+def get_headers(assignments):
+    headers = ['Email', 'SID']
+    new_assignments = []
```
Comment (colinschoen) [inline server/jobs/export_grades.py]: This doesn't seem to be used.

**Comment by epai** [comment]
Whoops, turns out `new_assignments` was necessary (just forgot to return it instead of "assignments")

**Comment by epai** [comment]
Great!  I'm going to try testing it out on staging to make sure it works, then I'll merge it in

**Comment by colinschoen** [comment]
@ericpai is this good to go?

**Comment by epai** [comment]
should be good to go, although it's a bit slow

**Pull request merged**